build/openresty/patches/nginx-1.25.3_07-fix-lua-context-clean-by-send-header.patch

```diff
@@ -0,0 +1,45 @@
+diff --git a/bundle/ngx_lua-0.10.26/src/ngx_http_lua_util.c b/bundle/ngx_lua-0.10.26/src/ngx_http_lua_util.c
+index 8fd2656..b2fdb6c 100644
+--- a/bundle/ngx_lua-0.10.26/src/ngx_http_lua_util.c
++++ b/bundle/ngx_lua-0.10.26/src/ngx_http_lua_util.c
+@@ -549,6 +549,10 @@ ngx_http_lua_send_header_if_needed(ngx_http_request_t *r,
+         if (!ctx->buffering) {
+             dd("sending headers");
+             rc = ngx_http_send_header(r);
++            if (r->filter_finalize) {
++                ngx_http_set_ctx(r, ctx, ngx_http_lua_module);
++            }
++
+             ctx->header_sent = 1;
+             return rc;
+         }
+diff --git a/bundle/ngx_lua-0.10.26/t/002-content.t b/bundle/ngx_lua-0.10.26/t/002-content.t
```

Comment:
We do not run openresty's test in kong, should we remove this part in patch?

Reply:
See: https://github.com/Kong/kong/pull/12940#discussion_r1583400706

Reply:
It does not matter if we remove or not remove it. But I think we must keep these changes in our patch to keep our repo sync with openresty upstream.